keypirinha-steam/src/steam.py

```diff
@@ -140,7 +140,7 @@ def get_applist(self, steam_dir):
 
         # Load appinfo.vdf to extract info about games
         with open(appinfo_path, 'rb') as fp:
-            _, steamapps = appcache.parse_appinfo(fp)
+            _, steamapps = appcache.parse_appinfo(fp, dict)
```

Comment:
is this needed? I think the default for mapper is `dict` down the line anyways?

Reply:
I thought so as well, but in my testing it didn't seem to work without specifying dict (got an error about an object not being a subclass)